ipatests/test_xmlrpc/test_pwpolicy_plugin.py

```diff
@@ -31,6 +31,56 @@
                                               Declarative)
 
 
+def pwpolicy_cmd(
+        cmd, minlife, cospriority=None,
+        pwpolicy_group=None):
+    """Helper method to add or modify the password policy
+
+    :param cmd: Either pwpolicy_add or pwpolicy_mod
+    :pram minlife: minimum life for the password to exist
```

Comment:
typo `pram`
minlife: The minimum amount of time (in hours) that must pass between two password change operations.